### PR TITLE
Add convenience methods for when empty labels are used.

### PR DESCRIPTION
**Pull request by dengliming**
for [https://github.com/open-telemetry/opentelemetry-java/issues/1533](https://github.com/open-telemetry/opentelemetry-java/issues/1533)
cc @kenfinnigan @jkwatson

### CODE REVIEW

File: api/src/main/java/io/opentelemetry/metrics/DoubleUpDownCounter.java [hunk 1]
```diff
@@ -67,6 +67,8 @@
    */
   void add(double increment, Labels labels);
 
+  void add(double increment);
```
Comment (jkwatson) [inline api/src/main/java/io/opentelemetry/metrics/DoubleUpDownCounter.java:70]: these methods all need javadoc, please

**Comment by jkwatson** [comment]
This seems like a good user-friendly API change. Can you add some tests for the SDK that assert that the no-label versions have the same result as the Labels.empty() versions? 

Thanks!

**Comment by dengliming** [comment]
> This seems like a good user-friendly API change. Can you add some tests for the SDK that assert that the no-label versions have the same result as the Labels.empty() versions?
> 
> Thanks!

sure. will fix it later. thanks

File: api/src/main/java/io/opentelemetry/metrics/DoubleUpDownCounter.java (hunk 1 above)
Comment (dengliming) [reply api/src/main/java/io/opentelemetry/metrics/DoubleUpDownCounter.java:78]: fixed at lastest commit.

File: api/src/main/java/io/opentelemetry/metrics/DoubleCounter.java [hunk 2]
```diff
@@ -64,6 +64,16 @@
...
 
+  /**
+   * Adds the given {@code increment} to the current value. The values cannot be negative.
+   *
+   * <p>The value added is associated with the current {@code Context} and with empty labels.
+   *
+   * @param increment the value to add.
+   * @since 0.8.0
+   */
+  void add(double increment);
```
Comment (anuraaga) [inline api/src/main/java/io/opentelemetry/metrics/DoubleCounter.java:75]: Not related to this PR at all - @jkwatson I was surprised to see here that we don't allow negative additions

Comment (jkwatson) [reply api/src/main/java/io/opentelemetry/metrics/DoubleCounter.java:75]: The UpDownCounters are the ones that allow negative increments.

Comment (anuraaga) [reply api/src/main/java/io/opentelemetry/metrics/DoubleCounter.java:75]: Ah got it! I think... do we need two types of counters? Just food for thought no strong opinion, mostly just used to things like `AtomicInteger` in terms of API.

Comment (jkwatson) [reply api/src/main/java/io/opentelemetry/metrics/DoubleCounter.java:75]: There was intense debate in the metric specs about this. The rationale (for good or ill) was that having different instruments that reflect the different semantics of the counting was valuable for instrumentation, to make it very clear what sort of counting you were doing. We went the opposite way from OpenCensus, which (I think?) has one instrument for everything, and lots of options.

**Comment by jkwatson** [comment]
I'm going to merge this. If @carlosalberto or @bogdandrutu have a problem with this small API change, we can revisit.

**Pull request merged**